news/4.34/platform.html

```diff
@@ -181,6 +181,39 @@ <h2>General Updates </h2>
         </p>
       </td>
     </tr>
+    
+    <tr id="edge-browser-improvements"> <!-- https://github.com/eclipse-platform/eclipse.platform.swt/issues/1466 -->
+      <td class="title">Improvements to Edge Browser (Windows only)</td>
+      <td class="content">
+        <p>
+          In an effort to make it the default browser for Windows, several issues that affected the Edge browser 
+          have been addressed in this release. Some of them are:
+          <ul>
+          	<li>Added support for basic authentication</li>
+          	<li>Inherited the proper light/dark theme from Eclipse instead of Windows</li>
+          	<li>Fixed obscure "No more handles" issues</li>
+          	<li>Fixed deadlocks that occurred due to the way Edge handles requests</li>
+          	<li>Fixed some styling issues in the welcome page for Eclipse</li>
+          	<li>Fixed some issues with javascript being used in an empty (local) page</li>
+          </ul>
+        </p>
+        <p>
+          Edge still <b>isn't</b> the default browser but it is mature enough that it can be activated and used already.
+          In order to replace IE with Edge as the browser of choice in Windows, you can either: 
```

Comment:
This does not change your Windows default browser but the one used by Eclipse.
```suggestion
          In order to replace IE with Edge as the browser of choice in Eclipse, you can either: 
```

Reply:
It's meant to be _In Eclipse while running on Windows_, but that's a mouthful. 
It's clear from the context that we are talking about the browser for Eclipse.

Reply:
I disagree here. The formulation is incorrect.

Reply:
I removed _in Windows_. It's clear from the context that we're talking about _Edge for Eclipse in Windows_.